examples/ReinterpretLog.hs

```diff
@@ -21,145 +25,161 @@
 {-# LANGUAGE TypeApplications #-}
 {-# LANGUAGE TypeOperators #-}
 {-# LANGUAGE UndecidableInstances #-}
+{- FOURMOLU_ENABLE -}
 
-module ReinterpretLog
-( example
-, application
-, runApplication
+module ReinterpretLog (
+  example,
+  application,
+  runApplication,
 ) where
 
-import Control.Algebra
+import Control.Algebra (Algebra (..), send, type (:+:) (..))
 import Control.Carrier.Reader
 import Control.Carrier.Writer.Strict
-import Control.Monad.IO.Class (MonadIO(..))
+import Control.Monad.IO.Class (MonadIO (..))
 import Data.Kind (Type)
 import Hedgehog
-import Prelude hiding (log)
 import Utils
+import Prelude hiding (log)
+
+{-
+## The application
+
+### Log message
 
---------------------------------------------------------------------------------
--- The application
---------------------------------------------------------------------------------
+Our structured log message. In this example, we just tag a `String` with its
+severity, but this can be anything.
+-}
 
--- Our structured log message. In this example, we just tag a 'String' with its
--- severity, but this can be anything.
 data Message
   = Debug String
   | Info String
 
--- Render a structured log message as a string.
+{- Render a structured log message as a string. -}
```

Comment:
I am really 👎🏻 on essentially all of the formatting changes in this file. They're needless, they make the diff harder to read, and they specifically do not reflect the formatting used elsewhere in the project.